automation/run-tests.sh

```diff
@@ -269,6 +269,19 @@ function vdsm_tests {
     cd -
 }
 
+function upload_coverage {
+    if [[ "$CI" == "true" ]] ;then
+        container_exec "
+            cd $CONTAINER_WORKSPACE &&
```

Comment:
There is no coveralls report in this PR, so this might not be working.

Reply:
coveralls has produced a link, just no comments added to this PR. No idea why.